### DISCUSSION TITLE
https://thanhle.blog/vi/blog/ki-nang-nua-cung-nua-mem-cho-dev

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#18] [Ideas]
# https://thanhle.blog/vi/blog/ki-nang-nua-cung-nua-mem-cho-dev

https://thanhle.blog/vi/blog/ki-nang-nua-cung-nua-mem-cho-dev

**Comment by nguyenhoaikhang37** [comment]
bài viết hay quá anh Thành ơi, văn viết lém lỉnh, có cả ảnh minh hoạ thú vị. Mong sắp tới anh làm thêm 1 bài viết nói về: Cách học một ngôn ngữ hay thư viện mới

**Reply by thanhlmm** [reply]
Haha. Thanks em, nice idea. Có thể bài sau anh sẽ viết về cái này

**Comment by djdaantruong** [comment]
Em đọc blog của anh xong thích lắm, anh viết đơn giản rõ ràng, đầy đủ ý, nhiều kiến thức hay, cho em biết và hiểu rõ thêm một số skill cần học để làm tốt trong học tập lẫn trong công việc sau này. Sắp tới khi nào rảnh rổi anh viết một bài chia sẻ về thực hành git nha, em rất thích git luôn ^^

**Reply by thanhlmm** [reply]
Thanks e!. Nhớ chờ vãi tuần nữa anh sẽ viết về phần này nhé!
